### ISSUE TITLE
Japanese word should not normalize synonyms during indexation.

### ISSUE THREAD

**Issue by miiton** [meilisearch/meilisearch#2159]
**Describe the bug**

Maybe Chinese and Korean are also ...

I tried add a synonym below

```json
{"東京スカイツリー駅": ["とうきょうスカイツリー駅"], "フォトショップ": ["photoshop"]}
```

then synonyms were added below

```json
{"フ ォ ト シ ョ ッ プ":["photoshop"],"東 京 ス カ イ ツ リ ー 駅":["と う き ょ う ス カ イ ツ リ ー 駅"]}
```

*Spaces are inserted between every chars.*

This gets stuck in Synonym's three-word limit, right?

> Multi-word synonyms are limited to a maximum of three words.
> [Synonyms \| Meilisearch Documentation v0\.25](https://docs.meilisearch.com/reference/features/synonyms.html#multi-word-phrases)

This issue means that Japanese propernouns cannot be registered as Synonym.
From #1184, I understood that non-latin characters are not normalized, is that wrong?

**To Reproduce**

1. add synonym

```bash
curl -X POST http://localhost:7700/indexes/jpdocs/settings/synonyms -H 'Content-Type: application/json' \
  --data-binary '{"東京スカイツリー駅": ["とうきょうスカイツリー駅"],"フォトショップ":["photoshop"]}'
```

2. get synonyms

```bash
curl -X GET http://localhost:7700/indexes/jpdocs/settings/synonyms -H 'Content-Type: application/json'
```

**Expected behavior**

```bash
curl -X GET http://localhost:7700/indexes/jpdocs/settings/synonyms -H 'Content-Type: application/json'
```

I expected.

```json
{"フォトショップ":["photoshop"],"東京スカイツリー駅":["とうきょうスカイツリー駅"]}
```

**Meilisearch version:  v0.25.2**

**Additional context**

- Synonyms are very useful in Japan because it is very common to use abbreviations and write English words in Katakana 😄
- Issues I saw before posting: #1184

**Comment by ManyTheFish** [comment]
Hey @miiton, Meilisearch doesn't support Japanese for now and your issue is highly related to it. I saw [your PR on the tokenizer](https://github.com/meilisearch/tokenizer/pull/70) and I will review it this week.
I think that your PR will fix it. 🙂

**Comment by curquiza** [comment]
I assign @ManyTheFish to check whether the new tokenizer version fixes or not this issue 🙂

**Comment by curquiza** [comment]
I've just tested and it works, will be released in v0.27.0 (first RC is coming next week).
Please tell me if you still have an issue with this
